tests/links_tests/model_tests/fpn_tests/test_faster_rcnn.py

```diff
@@ -31,24 +31,35 @@ def __call__(self, x):
 
 class DummyFasterRCNN(FasterRCNN):
 
-    def __init__(self, n_fg_class):
+    def __init__(self, n_fg_class, min_size, max_size):
         extractor = DummyExtractor()
         super(DummyFasterRCNN, self).__init__(
             extractor=extractor,
             rpn=RPN(extractor.scales),
             head=Head(n_fg_class + 1, extractor.scales),
+            min_size=min_size, max_size=max_size,
         )
 
 
 @testing.parameterize(
-    {'n_fg_class': 1},
-    {'n_fg_class': 5},
-    {'n_fg_class': 20},
+    {'n_fg_class': 1, 'min_size': 200, 'max_size': 400, 
+     'in_shape': (3, 200, 50), 'expected_shape': (3, 400, 100)},
+    {'n_fg_class': 1, 'min_size': 800, 'max_size': 133, 
+     'in_shape': (3, 480, 640), 'expected_shape': (3, 800, 1088)},
```

Comment:
`prepare` can accept a batch of images of different shapes. 
How about `'in_sizes': [(480, 640), (320, 320)], 'expected_size': (800, 1088)` ? (I omit `3` because it is not parameterized)